board/post-image.sh

```diff
@@ -90,6 +90,11 @@ __EOF__
 			cat "$BR2_EXTERNAL_PICAM_PATH/camera.txt" >> "${BINARIES_DIR}/camera.txt"
 		fi
 
+		# Add default resolutions.txt if it exists
+		if [ -f "$BR2_EXTERNAL_PICAM_PATH/package/piwebcam/resolutions.txt" ]; then
```

Comment:
I don't think this should be an optional file, like `camera.txt` is. We could certainly move this file to `/etc`.

In that case I *think* you could move that line to `package/piwebcam/piwebcam.mk`?

Reply:
I could do that, however the aim of this PR was to make the output resolutions configurable without having to build the whole image.

One could put the default file into `/etc` and make it possible to overwrite it with another file in `/boot` however I see no advantage in there.

Reply:
Well, we get a new build with each PR/commit so this shouldn't be an issue :)

The advantage would be that configuration settings shouldn't belong inside shell files. Furthermore /boot is writeable, whereas /etc is not.